web/app/tournament/tourny.py

```diff
@@ -169,3 +172,93 @@ def select_player(tournament_id, tournament_player_id):
         # flash("Could not delete player: {{ tp.name }}")
         return redirect(url_for('.players_index',
             tournament_id=tournament_id))
+
+
+@tournament.route('/<tournament_id>/matches/', methods=['GET', 'POST'])
+def matches_index(tournament_id):
```

Comment:
I know we've talked about this from time to time, but I was doing some rea

http://stackoverflow.com/questions/34214807/python-flask-different-render-template-for-get-and-post-methods-possible